lighthouse-core/report/html/renderer/report-renderer.js

```diff
@@ -207,12 +207,14 @@ class ReportRenderer {
     }
 
     // Fireworks
-    const scoresAll100 = report.reportCategories.every(cat => cat.score === 1);
-    if (scoresAll100) {
-      headerContainer.classList.add('score100');
-      this._dom.find('.lh-header', headerContainer).addEventListener('click', _ => {
-        headerContainer.classList.toggle('fireworks-paused');
-      });
+    if (!this._dom.isDevTools()) {
```

Comment:
is this a pavel-reaction undoing? 😆 

or do we just not want to deal with the differences right now getting it to work in a smaller header?

Reply:
we are brushing up on the release deadline for next chrome, so quick is good. but also we don't even have a design for fireworks in DT. @exterkamp should totes look into that